util/testutil/error.go

```diff
@@ -13,15 +13,11 @@
 
 package testutil
 
-// ErrorEqual compares Go errors for equality.
-func ErrorEqual(left, right error) bool {
-	if left == right {
-		return true
-	}
-
+// ErrorsEqual fails when Go errors are not equal.
+func ErrorsEqual(tb TB, left, right error) {
```

Comment:
It might be useful to also rename the the parameters to `expected, actual error`

Reply:
I renamed it to `exp` and `act` just like in `Equals` function.